### ISSUE TITLE
Switch from pull model to push model in Storage

### ISSUE THREAD

**Issue by MaxiBoether** [eth-easl/modyn#99]
Currently, the supervisor regulary queries the Storage for now datapoints.

@anakli asked us to in the long term switch to a push model where the Storage informs the Supervisor about updates instead, to reduce latency.

This is not a priority for now.

**Comment by vGsteiger** [comment]
Currently waiting on #249 with the implementation of this.

**Comment by vGsteiger** [comment]
Waiting on #270

**Comment by MaxiBoether** [comment]
I think we can close this issue for now. I don't see the benefit of pushing over pulling, since the overhead is quite small and gRPC matches the pull pattern more.
